packages/components/src/utils/getTransformedPageContent.ts

```diff
@@ -31,11 +31,18 @@ export const getTransformedPageContent = (content: IsomerComponent[]) => {
           }
         }),
       }
-
-      transformedContent.push(transformedBlock)
+    } else if (
+      block.type === "infocards" ||
+      block.type === "infocols" ||
+      block.type === "infopic" ||
+      block.type === "keystatistics"
```

Comment:
feels like this should already be a type or type guard

Reply:
Wait checking here, this one is for these specific blocks where we want to add the ID but the function will be able to take any block. We don't necessarily want to add the ID for blocks that does not support having anchors in them (e.g. infobar).

Reply:
yea, but this checks for blocks that can have toc - so we can write a typeguard that accepts a string and narrows it to these 4 specifically after. 

only noting it here because this check is also repeated in `getTableOfContents`